paywall/src/components/checkout/Checkout.tsx

```diff
@@ -8,17 +8,17 @@ import CheckoutLock from './CheckoutLock'
 interface Props {
   locks: Locks
   config: PaywallConfig
-  account: Account
+  account: Account | null
```

Comment:
Just making sure I understand, it looks like `Account` is passed down through these solely to use in constructing `NoKeyLock`. Is `Account` something that `NoKeyLock` could fetch on its own (from state, etc.)?

Reply:
we aren't using redux in the checkout component, so all state is passed down through props. Otherwise, the answer would have been yes :)